src/libLocalDescriptorCache/DescDbProxy.cpp

```diff
@@ -350,12 +350,27 @@ void DescDbProxy::getDescriptors(const std::string &f_hostAddress)
     }
 }
 
+grpc::Status DescDbProxy::closeDescDbStream(std::optional<std::chrono::time_point<std::chrono::system_clock>> deadline)
+{
+    if( m_disableCache  == false )//cache enabled, no reflection stream required.
```

Comment:
this is a slippery slope.
If cache is enabled, we will still use the reflection stream, if the cache entry is outdated or if the cache does not hold data for the host.
see line 330

Reply:
ah I see it. I change the condition to give prefence to m_reflectionDescDb check. if the ptr is valid we close the stream any how.